linera-client/src/storage.rs

```diff
@@ -193,29 +195,38 @@ example service:tcp:127.0.0.1:7878:table_do_my_test"
         if let Some(s) = input.strip_prefix(ROCKS_DB) {
             if s.is_empty() {
                 return Err(Error::Format(
-                    "For RocksDB, the formatting has to be rocksdb:directory:namespace".into(),
+                    "For RocksDB, the formatting has to be rocksdb:directory or rocksdb:directory:spawn_mode:namespace".into(),
                 ));
             }
             let parts = s.split(':').collect::<Vec<_>>();
             if parts.len() == 1 {
                 let path = parts[0].to_string().into();
                 let namespace = DEFAULT_NAMESPACE.to_string();
-                let storage_config = StorageConfig::RocksDb { path };
+                let spawn_mode = RocksDbSpawnMode::SpawnBlocking;
```

Comment:
Shouldn't we auto-detect here?

Reply:
I do not think so. That introduces unpredictability.
But I added the option "runtime" so users can use that directly if they so want.